### PR TITLE
Desktop_version PR

### PR DESCRIPTION
**Pull request by sibomana123**
For this project I used knowledge i got in the previous week and I used to make a good desktop version which good responsiveness 
I have applied css flexbox and CSS grid  , In this version the minimum size of the screen is 768px 

NB: Before of this I had a desktop_brnch so by accident I made a merge ; so according to The last 3 reviewers I was trying to make the change according to what they tell me now it's collected

### CODE REVIEW

**Comment by sibomana123** [comment]
Please! This is my last extra review in this first module😅

File: index.html [hunk 1]
```diff
@@ -96,19 +94,18 @@ <h1 class="descr__header">Multi-Post Stories</h1>
...
-          
-          <img class="post__image" src="img/Snapshoot Portfolio.svg" alt="">
+        </div>        
+
+      </div>
+      
+      <div class="post__div">
+        <div class="post__side__img">
+          <img class="post__img__mobile" src="img/Snapshoot Portfolio (3).svg" alt="">
         </div>
```
Comment (bobb-Rob) [inline index.html:109]: - [ ] I  must confess your desktop design is excellent and beautiful 👏🏾 👏🏾 Well done!

But you actually traded your mobile design for the desktop. Your project in mobile view is a big mess. see below the state. It is expected that you use media queries to adapt what you already did well on mobile to desktop. I suggest that you revert your code to your last commit where your mobile view was displayed correctly thereafter you slowly use media queries to adapt it to the desktop without distorting the mobile view design. I noticed that you have a file called `desktopversion.css`, what about the `mobileversion.css`?

Your project in mobile view 👇🏾 

image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @sibomana123, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

 ## Required Changes ♻️

- [ ] Please do not mistakenly merge this pull request until you fix this issue with mobile view.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sibomana123** [comment]
@bobb-Rob Hello brother, thanks for what you told me but before I did it and include the mobile version css but first reviewer refused even instructions and asked to work on a desktop  >=768px  right now I haven't the ability to request a review how can do it?

**Comment by bobb-Rob** [comment]
Could you fix the issue and ensure it is fixed, then reach out to student success, they will give you another chance—best of luck. 

_Ensure your application looks very good on both views, mobile and desktop._

File: small_version.css [hunk 2]
```diff
@@ -0,0 +1,549 @@
+@media screen and (max-width: 375px) {
```
Comment (bellom) [inline small_version.css]: You need to set the media query to cover a minimum width of 375px and a max 768px as stated in [mobile milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=Minimum%20mobile%20screen,should%20be%20768px).

Right now your webpage breaks on screens greater than 375px 🤦‍♂️
image

To fix this;
Try to set the min-width 375px and max-width 768px in media query - [documentation](https://www.freecodecamp.org/news/media-queries-width-ranges/)

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @sibomana123,

### This is a manual review

## Request Changes :+1:

You did a great working on the previous request change, however, your webpage breaks on screens greater than 375px.

Check the inline comment below for hint on how to solve this.

Tag me once you're done - I will re-review.

Cheers and Happy coding!:clap::clap::clap:

File: small_version.css (hunk 2 above)
Comment (sibomana123) [reply small_version.css]: Hi @bellom,

I was trying to make my portfolio to be responsive on all screens, could you check for me?

Comment (bellom) [reply small_version.css]: Hey @sibomana123, 

I can see you made some changes - you only need one-liner of code.

kindly revert what you have done and try this - `@media screen and (min-width: 375px) and (max-width: 768px) {`

**Comment by bellom** [comment]
@sibomana123 check my comment above.

File: small_version.css (hunk 2 above)
Comment (sibomana123) [reply small_version.css]: Hi @bellom ,

Thank you for your vast and guidance to me ,  think  was trying to correct could you check for me ?

Comment (bellom) [reply small_version.css]: @sibomana123 revert to this commit - [06f0c34](https://github.com/sibomana123/MV__Portfolio/pull/6/commits/06f0c34d00128efcae018a48b40d4e27a49d5ad2) before you add the hint above.

**Comment by bellom** [comment]
Hi @sibomana123,

### This is a manual review

# Approved :+1:

Cheers and Happy coding!:clap::clap::clap:

File: small_version.css (hunk 2 above)
Comment (sibomana123) [reply small_version.css]: @bellom  thanks for support

**Pull request merged**